src/crx.js

```diff
@@ -187,6 +188,11 @@ ChromeExtension.prototype = {
 	      throw new Error('crx.load needs to be called first in order to prepare the workspace.');
       }
 
+      var crxignore = null
+      try {
+        crxignore = gitignoreParser.compile(fs.readFileSync(selfie.path + '/.crxignore', 'utf8'))
```

Comment:
Also, we do not want silent try catch, nor sync operations.

Best would be:

- feeding ignore patterns from the CLI (eg: `crx --ignore */*.js --ignore **/.*` ... )
- fetching the ignore rules form a `.crxignore` file indeed (but you want the result to be a list of ignored files) (eg: `crx --ignore-file .crxignore --ignore-file another/.crxignore`)

Both would be provided as a parameter to the class itself.

I reckon this PR addresses the `--ignore-file` only but you get the idea.